### DISCUSSION TITLE
Toggle "stockpiling box" icon for "enable stockpiling" button

### DISCUSSION THREAD

**Discussion by Ouaz** [Keriew/augustus#490] [Ideas]
Add a wooden box icon for "enable stockpiling" button in farms and raw material buildings panels (related to #403 )
_______________________
Here's the stockpiling box icon (transparent png file 12x12px) : ![stockpiling_box_icon](https://user-images.githubusercontent.com/11946570/134412708-a4d6ebbf-3b8a-4bf9-9114-fd3ccb9f4298.png)
_______________________
**Wheat farm "Stockpiling disabled"** ---------------------->**Wheat farm "Enable stockpiling to a warehouse"**

<img src="https://user-images.githubusercontent.com/11946570/134412941-e8b190f6-af05-40f3-bee5-b279d4abe839.png" alt="stockpiling_off_wheat_farm" width="350"/> ![stockpiling_on_wheat_farm](https://user-images.githubusercontent.com/11946570/134413537-4523e6de-5867-4237-8ae9-a5e8d70c27b0.png)

Let me know if it's ok, or if it needs to be improved. :stuck_out_tongue: 

Thanks in advance for the answer.

**Comment by Keriew** [comment]
I think, it looks great, thanks for it! Added it in 08d5239
